CONTRIBUTING.md

```diff
@@ -154,45 +154,57 @@ toolkit for interacting with AWS backend resources.
 
 ## Integration Tests
 
-In addition to unit tests which mock Amplify API interaction, this repository has integration tests which 
-test functionality with real Amplify backends. The integration test script will execute tests in example 
+In addition to unit tests which mock Amplify API interaction, this repository has integration tests which
```

Comment:
The formatting changes in this file are auto formatting from prettier. The changes generally seem to be an improvement, and the majority of the file was already following that formatting, so I decided to leave them.

Pro tip: use https://github.com/aws-amplify/amplify-flutter/pull/910/files?diff=split&w=1 to ignore the whitespace changes

I also updated a few typos in the file.